llvm/lib/Target/X86/X86MCInstLower.cpp

```diff
@@ -350,8 +350,11 @@ MCOperand X86MCInstLower::LowerMachineOperand(const MachineInstr *MI,
     return MCOperand::createImm(MO.getImm());
   case MachineOperand::MO_MachineBasicBlock:
   case MachineOperand::MO_GlobalAddress:
-  case MachineOperand::MO_ExternalSymbol:
-    return LowerSymbolOperand(MO, GetSymbolFromOperand(MO));
+  case MachineOperand::MO_ExternalSymbol: {
```

Comment:
did you mean for MO_GlobalAddress and MO_MachineBasicBlock to also hit this branch?

Reply:
welp, no, 🤦‍♀️,  thanks for point this out!